modules/backend/classes/BackendController.php

```diff
@@ -49,6 +49,26 @@ class BackendController extends ControllerBase
      */
     public function __construct()
     {
+        // Find requested controller to determine if any middleware has been attached
+        $pathParts = explode('/', Request::path());
```

Comment:
Does this support installations that are in subfolders? (i.e. not attached to the root domain)

Reply:
It should now. :)